src/routes/mod_creation.rs

```diff
@@ -113,8 +115,20 @@ struct ModCreateData {
     pub source_url: Option<String>,
     /// An optional link to the mod's wiki page or other relevant information.
     pub wiki_url: Option<String>,
+    /// An optional link to the mod's license page
+    pub license_url: Option<String>,
+    /// An optional link to the mod's discord.
+    pub discord_url: Option<String>,
     /// An optional boolean. If true, the mod will be created as a draft.
     pub is_draft: Option<bool>,
+    /// The support range for the client mod
+    pub client_side: SideType,
+    /// The support range for the server mod
+    pub server_side: SideType,
+    /// The license id that the mod follows
+    pub license_id: String,
```

Comment:
If licenses are strictly limited to the options we give, there needs to be an "other" category and a way of linking to the license.  People will always come up with weird licenses, so we can't enumerate them all.

Reply:
I'll add a "Custom License" license in the migration for the sake of this